coordinator/internal/authority/userapi_test.go

```diff
@@ -253,32 +253,35 @@ func TestRecovery(t *testing.T) {
 	var seed [32]byte
 	var salt [32]byte
 	testCases := []struct {
-		name     string
-		seed     []byte
-		salt     []byte
-		wantCode codes.Code
+		name        string
+		seed        *[]byte
+		salt        *[]byte
+		wantCode    codes.Code
+		wantMessage string
 	}{
 		{
-			name:     "empty seed",
-			salt:     salt[:],
-			wantCode: codes.InvalidArgument,
+			name:        "empty seed",
+			seed:        toPtr[[]byte](nil),
```

Comment:
The `toPtr` is superfluous here, since type of the member variable should make nil already to a typed nil.

Reply:
There's a difference: I'm creating a non-nil pointer to a nil slice here, which makes the test body override the seed with a nil slice.

Reply:
Makes sense, but I think there's still another way of testing the emptiness test cases with initialized 0 length slices and then using the nil slice to decide whether to replace or not. If you want to explicitly test nil inputs then I'd create different test cases (one with nil and one with zero length slices), if not then I'd go with just zero length slices.

Reply:
I explicitly want to test with nil slices, there is a test with a short slice already.

Reply:
Alright, good point